release-notes/7.0/preview/api-diff/preview2/Microsoft.NETCore.App/7.0-preview2_Microsoft.Win32.SafeHandles.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.Win32.SafeHandles
+
+``` diff
+ namespace Microsoft.Win32.SafeHandles {
+     public abstract class SafeNCryptHandle : SafeHandleZeroOrMinusOneIsInvalid {
+-        public override bool IsInvalid { get; }
```

Comment:
What is the purpose of our publishing API diffs?  Assuming it's to help developers understand meaningful differences since the last release, the removal of an override of a virtual on an abstract class has no impact on developers and shouldn't be included. (On top of that, I don't even see this override in previous releases.)

Reply:
I don't understand the argument of it not being helpful to know that an override was removed. It sounds exactly like one the purposes of the API diff PR.

This class was moved from `System.Security.Cryptography.Cng` to `System.Security.Cryptography`.

Before (release/7.0-preview1), the `IsInvalid` property was there: https://github.com/dotnet/runtime/blob/405337939cf8d793661f6c6c0007c3f519b26f0d/src/libraries/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs#L13

After (release/7.0-preview2), the `IsInvalid` property was removed: https://github.com/dotnet/runtime/blob/e24f66dff0770eee344038da8c12476d8c450c41/src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs#L9-L17